runtests.py

```diff
@@ -43,6 +52,7 @@
             'ENGINE': 'django.db.backends.sqlite3',
         }
     },
+    MIGRATION_MODULES=DisableMigrations(),
```

Comment:
Might be a dumb question, but why not just `{}`?

Reply:
{} is the default, which would lead us back to square one with the original issue